service/src/test/java/bio/terra/cbas/controllers/VerifyPactsAllControllers.java

```diff
@@ -76,6 +76,20 @@ class VerifyPactsAllControllers {
   // This mockMVC is what we use to test API requests and responses:
   @Autowired private MockMvc mockMvc;
 
+  // mock objects
+  UUID fixedMethodVersionUUID = UUID.fromString("90000000-0000-0000-0000-000000000009");
```

Comment:
Just checking... is this assuming the current prestaged database state? Or just a very similar looking UUID?

Reply:
If I understand correctly, there's no database being used during pact verification. It's probably worth changing this UUID to one that is more obviously different

Reply:
I think this is just a very similar looking ID. It is fixed at `90000000-0000-0000-0000-000000000009` because that's what the consumer tests expect in the response. For example [here](https://github.com/DataBiosphere/cbas-ui/blob/160454ace85d48024adfa9cd8856bfeb965e376f/src/libs/ajax.test.js#L128).